src/Attributes/Enums/UnixTimeResolution.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Crell\Serde\Attributes\Enums;
+
+enum UnixTimeResolution: int {
+    case Seconds = 1;
+    case Milliseconds = 1_000;
+    case Microseconds = 1_000_000;
+    case Nanoseconds = 1_000_000_000;
```

Comment:
I honestly won't bother with nanoseconds at all. You're not doing particle physics with PHP.

Reply:
Like I said here: https://github.com/Crell/Serde/pull/51#discussion_r1486898165, we don't always get to choose what systems and formats we are interacting with. Otherwise, I agree 100%. In other words, this resolution probably isn't for PHP, but interacting with systems written in Go (which uses nanoseconds) or Python (which also uses nanoseconds).